index.css

```diff
@@ -55,10 +54,27 @@ body {
 	height:9rem;
 }
 
+.Grid-description,
+.Grid-booking,
+.Grid-calendar,
+.Grid-infoLink,
+.Grid-video{
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.Grid-description-text,
+.Grid-booking-text,
+.Grid-calendar-text,
+.Grid-infoLink-text,
+.Grid-video-text {
```

Comment:
Instead of repeating all the selectors, where it is easy to forget one or misspell one, it can (sometimes) be easier with a new, single class you reuse.